manubot/process/util.py

```diff
@@ -33,6 +33,7 @@
     is_valid_citekey,
     standardize_citekey,
 )
+from manubot.filters import MB_FILTERS
```

Comment:
we should probably move this module to `manubot.process.jinja`. Probably shouldn't be a top-level submodule.

Reply:
I think that's a new module. If I understand correctly - done 👍